bz-manage.sh

```diff
@@ -8,7 +8,17 @@ fi
 get_bzs() {
   # shellcheck disable=SC2086
   # jq arguments $3 are unquoted to allow full expansion
-  bugzilla query --from-url "$1" --json | jq ${3:-} "${2:-.bugs[].id}"
+  local url jq_expression
+  url="$1"; shift
+  if [ "${RELEASE_PLUS:-false}" = true ]; then
+    url="${url}&f1=flagtypes.name&o1=substring&v1=release%2B"
```

Comment:
What if $url already contains some f1,o1,v1 parameters, will this work? Like with https://bugzilla.redhat.com/buglist.cgi?limit=100&component=rhel-system-roles&cf_internal_target_release=9.3.0&f1=flagtypes.name&f2=external_bugzilla.url&f3=ext_bz_bug_map.ext_bz_bug_id&o1=substring&o2=substring&o3=anywordssubstr&v1=requires_doc_text?&v2=github&v3=pull

Reply:
fixed in latest commit